op-challenger/game/fault/trace/vm/executor.go

```diff
@@ -77,37 +82,19 @@ func (e *Executor) DoGenerateProof(ctx context.Context, dir string, begin uint64
 		"--input", start,
 		"--output", lastGeneratedState,
 		"--meta", "",
-		"--info-at", "%" + strconv.FormatUint(uint64(e.cfg.InfoFreq), 10),
+		"--info-at", "%" + strconv.FormatUint(uint64(e.cfg.Cfg().InfoFreq), 10),
```

Comment:
This just screams to me that pushing the global `Config` into `VmConfig` is the wrong move - it breaks the [law of demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) - admittedly it's more like a guideline really, but this is a pretty classic example of breaking it.